scripts/populate_prices.py

```
@@ -314,149 +318,226 @@ async def fetch_steam_price_single(
-                                params={"key": ITAD_API_KEY, "country": "US"},
-                                api_type="itad",
-                            )
+                for shop_query, itad_id in lookup_data.items():
+                    app_id = shop_query.replace("app/", "")
+                    if itad_id:
+                        new_mappings[app_id] = itad_id
+                    else:
+                        failed_appids.append(app_id)
+
+                # Any in chunk not in response are failures
+                resolved_in_chunk = {q.replace("app/", "") for q in lookup_data.keys()}
```

Ruff (0.15.7)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)